src/data-display/tables/data-table/PDataTable.vue

```diff
@@ -35,7 +35,7 @@
                                         <span class="th-contents"
                                               :class="{
                                                   [field.textAlign || DATA_TABLE_CELL_TEXT_ALIGN.left]: true,
-                                                  'has-icon': sortable && field.sortable || colCopy,
+                                                  'has-icon': (sortable && isFieldSortable(field.sortable)) || colCopy,
```

Comment:
How about changing the way to check sortable like this?

```suggestion
                                                  'has-icon': (isFieldSortable(field.sortable) || colCopy,
```